src/ec/packages/ec-component-expandable/expandable.story.js

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import demoData from './demo/data';
+
+import expandable from './expandable.html.twig';
+import notes from './docs/expandable.md';
+
+// Add SVG icon path.
+demoData.button.icon.path = defaultSprite;
+
+storiesOf('Components/Expandable', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'default',
+    () => {
+      const html = expandable({
+        label_expanded: text('Label Expanded', demoData.labelExpanded),
```

Comment:
as label is injected va js, the knobs do not work here.
I checked on ECL, we don't use knobs for expandable labels. So I guess we could just remove them.

Reply:
I managed to fix this by adding the storybook's methods in the `demoData` itself. If it's an issue to have knobs which were not there before in the React version, I can remove them.